packages/gatsby-theme-carbon/src/components/markdown/Markdown.module.scss

```diff
@@ -93,6 +93,26 @@
   color: $carbon--white-0;
 }
 
+//spacing for content within list item
+.list-item > * + :global(.bx--row),
+.list-item > * + :global(.gatsby-resp-image-wrapper),
+.list-item > * + pre {
+  margin-bottom: 1.5rem;
+}
+
+//second child in li needs top margin to separate it from li text
+.list-item > :global(.bx--row):nth-child(2),
+.list-item > :global(.gatsby-resp-image-wrapper):nth-child(2),
+.list-item > pre:nth-child(2) {
+  // 24px (1.5rem) - 8px of margin-bottom from paragraph above it (see line 72)
```

Comment:
**pedantic warning** generally don't want to use line numbers in comments since they can become outdated quickly.